src/web/functional/src/test/java/org/geogig/web/functional/FunctionalTestContext.java

```diff
@@ -57,7 +57,7 @@
     /**
      * A temporary folder to use for tests
      */
-    protected TemporaryFolder tempFolder = null;
+    public TemporaryFolder tempFolder = null;
```

Comment:
There is a function in this class called getTempFolder() that the code in the plugin can use.